cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1030,5 +1030,24 @@ function Verify-WorkerServices {
   Log_Todo "run more verification commands."
 }
 
+# Add a registry key for docker in EventLog so that log messages are mapped
+# correctly.
+# TODO: Fix this in the base image.
+function Create-DockerRegistryKey {
+  $tmp_dir = 'C:\tmp_docker_reg'
+  New-Item -Force -ItemType 'directory' ${tmp_dir} | Out-Null
+  $reg_file = 'docker.reg'
+  Set-Content ${tmp_dir}\${reg_file} `
+'Windows Registry Editor Version 5.00
+ [HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\EventLog\Application\docker]
+"CustomSource"=dword:00000001
+"EventMessageFile"="C:\\Program Files\\docker\\dockerd.exe"
+"TypesSupported"=dword:00000007'
+
+  Log-Output "Importing registry key for Docker"
+  reg import ${tmp_dir}\${reg_file}
```

Comment:
I think you already tried this but just to make sure: this succeeds when done multiple times, right?

Reply:
Yes, it always return `The operation completed successfully.`